admin/app/javascript/solidus_admin/web_components/solidus_select.js

```diff
@@ -64,6 +36,41 @@ class SolidusSelect extends HTMLSelectElement {
         break;
     }
   }
+
+  getTomSelectSettings() {
```

Comment:
Woops. The comment I made oin this file went missing somehow (thanks github. react is not an improvement btw...)

We could use a JS getter here.

```suggestion
  get tomSelectSettings() {
```

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get

Reply:
thanks I completely forgot about js getters